analyses/IA_cd_2010/doc_IA_cd_2010.md

```diff
@@ -0,0 +1,24 @@
+# 2010 Iowa Congressional Districts
```

Comment:
can you update this to follow the template here?

https://github.com/alarm-redist/fifty-states/blob/178a858ce6d34c356cfad1e050825c1aca0d1e1b/analyses/IA_cd_2020/doc_IA_cd_2020.md